.github/workflows/ci.yml

```diff
@@ -14,33 +14,33 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ '7.4', '8.2' ]
+                php-version: [ '8.1', '8.4' ]
                 db-type: [ sqlite, mysql, pgsql, agnostic ]
                 symfony-version: [ '5-min', '5-max', '6-min', '6-max', '7-min', '7-max']
                 exclude:
                   - symfony-version: '6-min'
-                    php-version: '7.4'
+                    php-version: '8.1'
                   - symfony-version: '6-max'
-                    php-version: '7.4'
+                    php-version: '8.1'
                   - symfony-version: '7-min'
-                    php-version: '7.4'
+                    php-version: '8.1'
                   - symfony-version: '7-max'
-                    php-version: '7.4'
+                    php-version: '8.1'
         env:
             DB_NAME: 'propel_tests'
             DB_USER: 'propel'
             DB_PW: 'propel'
         steps:
             - name: Install PostgreSQL latest
-              if: matrix.db-type == 'pgsql' && matrix.php-version != '7.4'
+              if: matrix.db-type == 'pgsql' && matrix.php-version != '8.1'
```

Comment:
I don't think this is needed - no need to exclude PHP 8.1 here, or any of the following instances of it either.  It may be required for the various Symfony versions, but those should be compared with packagist.org to find out min PHP version support.

Reply:
The idea is probably to run min PHP version with min Postgres (9 apparently), and test max PHP version against latest Postgres. Same as above, as long as it runs once I think it's fine - the code does not change with PHP/Postgres version.

Reply:
Yea, I wasn't thinking about us just running min/max tests, so this comparison makes more sense in light of that.